src/main/java/chat/Ui.java

```diff
@@ -0,0 +1,119 @@
+package chat;
+
+import java.util.Scanner;
+
+import chat.task.Task;
+
+/**
+ * Ui is a class that deals with interactions with the user.
+ */
+public class Ui {
+    
+    private static String greetingCat = "(=^. .^=)";
+    private static String goodByeCat = "(=^. .^=*)";
+    private static String goCat = "(=^. .^=)~~";
+    private static String gdJobCat = "\\(=^> <^=)/";
+    private static String errorCat = "(=^> <^=)'''";
+    
+    private static Scanner sc = new Scanner(System.in);
+
+    /**
+     * Prints lines that divide each response.
+     */
+    public void lines() {
```

Comment:
Perhaps naming the method as printLine() since it is not operating on a collection of objects (not lines but rather only one line) and use verb at the start also?